README.md

```diff
@@ -114,6 +114,28 @@ And a config like this:
 `styles/custom.css`, `styles/another.css`, and `remote.css` get included in the layout.
 `default.staticFiles` is the build-in jsdoc way of copying extra files.
 
+### ignore default theme css
```

Comment:
I think It is better to specify the file you want this attribute to contain rather than the name "Ignore". For example,

```json
"templates": {
    "customDefaultTheme": "path/to/some/theme.css",
    "customDefaultLayout": "path/to/some/layout.css",
    "scripts": [
        "path/to/some/script1",
        "path/to/some/script2"
    ]
}
```

```js
<?js if (templates.customDefaultTheme){ ?>
    <link type="text/css" rel="stylesheet" href="<?js= templates.customDefaultTheme ?>">
<?js } ?>

<?js if (templates.customDefaultLayout){ ?>
    <link type="text/css" rel="stylesheet" href="<?js= templates.customDefaultLayout ?>">
<?js } ?>
```

Reply:
good point - i'll update it shortly